packages/ethereum/contracts/HoprChannels.sol

```diff
@@ -536,7 +536,7 @@ contract HoprChannels is IERC777Recipient, ERC1820Implementer {
      * See https://ethresear.ch/t/you-can-kinda-abuse-ecrecover-to-do-ecmul-in-secp256k1-today/2384
      * @param response response that is used to recompute the challenge
      */
-    function computeChallenge(bytes32 response) public pure returns (address)  {
+    function _computeChallenge(bytes32 response) public pure returns (address)  {
```

Comment:
this doesn't need to be public, see `_getEncodedTicket`

Reply:
changed